ProjectFinderApi/V1/Controllers/UserController.cs

```diff
@@ -46,5 +47,22 @@ public IActionResult CreateUser([FromBody] CreateUserRequest request)
                 return UnprocessableEntity(e.Message);
             }
         }
+
+        /// <summary>
+        /// Get all users
+        /// </summary>
+        /// <response code="404">No users found.</response>
```

Comment:
Would be good to add all responses that are possibly returned by the controller, such as 500 and 200